src/app/js/admin/Appbar/ClearDialog.js

```diff
@@ -61,10 +59,8 @@ const ClearDialogComponent = props => {
         }
     }, [succeeded]);
 
-    const getInstanceName = () => /\/\/([a-z0-9-]+)./.exec(baseUrl)[1];
-
     const handleChangeField = e => {
-        const instanceName = getInstanceName();
+        const instanceName = extractTenantFromUrl(window.location.href);
```

Comment:
The two calls to extractTenantFromUrl can be factorised iat the begening of this componant